network/marks.csv

```diff
@@ -75,6 +75,7 @@ LKTA,LI,no,Lake Taylor,-42.783371186,172.266334114,712.9664,NZGD2000,2004-02-01T
 LOK1,XX,no,Glen Orkney Station Temporary,-41.859531782,173.818372429,678.4554,WGS84,2016-11-15T00:00:00Z,2017-11-02T00:00:00Z
 LOOK,CG,no,Mount Lookout,-42.01157,173.48244,1609,WGS84,2017-03-28T00:00:00Z,9999-01-01T00:00:00Z
 LRR1,XX,no,Clarence River Temporary,-42.144411066,173.815293346,921.9091,WGS84,2016-11-15T00:00:00Z,2017-05-22T00:00:00Z
+LY2T,GN,no,Lyttelton,-43.60663471,172.7236303,16.972,NZGD2000,2024-08-22T00:00:00Z,9999-01-01T00:00:00Z
```

Comment:
For now this network code is fine, but we have to update it as soon as we create a new one, specific for GNSS@TG sites